packages/client/src/Views/GameSetupNew.jsx

```diff
@@ -202,8 +205,11 @@ const AdminGameSetup = () => {
       const template = forceTemplate
       template.name = id
       template.uniqid = id
-      template.roles.map(role => role.roleId = getUniquePasscode(forces.forces, "p"))
-      await dispatch(saveForce(currentWargame, id, template, id))
+      template.roles.map(role => {
```

Comment:
hmm, doesn't `map` return a new array?  The returned results aren't being processed here. Shouldn't it be a `foreach` array function call?
(please teach me better JS :-D ).

Reply:
I just fix the lint checking :D 
![image](https://user-images.githubusercontent.com/26705621/149510987-844a8368-c3f7-408e-9f68-8a412200a4ca.png)